sbt-dumpschema-plugin/src/main/scala/org/virtuslab/akkasaferserializer/model/Codecs.scala

```diff
@@ -0,0 +1,10 @@
+package org.virtuslab.akkasaferserializer.model
```

Comment:
Nit: I wouldn't keep the codecs in `model` package... I'd rather keep them in wherever e.g. `SchemaWriter` lives